src/python_be.cc

```diff
@@ -1759,6 +1767,26 @@ ModelState::ModelState(TRITONBACKEND_Model* triton_model)
   }
 }
 
+std::string
+ModelState::DefaultArtifactName()
```

Comment:
Is there a way that this could be managed by the platform_model.py and not the python_be?

Ideally the python_be wouldn't need to know any details of how the platform_model_loader works

Reply:
I can push the logic of getting ArtifactName into platform model. We would need to parse the default_model_filename.